supportedBackends/google/pipelines/v1alpha2/src/main/scala/cromwell/backend/google/pipelines/v1alpha2/PipelinesConversions.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.backend.google.pipelines.v1alpha2
+
+import com.google.api.services.genomics.model.{Disk, LocalCopy, PipelineParameter}
+import cromwell.backend.google.pipelines.common.io.PipelinesApiAttachedDisk
+import cromwell.backend.google.pipelines.common.{PipelinesApiFileInput, PipelinesApiFileOutput, PipelinesApiInput, PipelinesApiLiteralInput}
+
+object PipelinesConversions {
+  implicit class EnhancedInput(val input: PipelinesApiInput) extends AnyVal {
+    def toGooglePipelineParameter: PipelineParameter = input match {
```

Comment:
This is a prime candidate for a typeclass, will standardize the def name on each of these types.

```scala
@typeclass trait ToPipelineParameter[A] { def toGooglePipelineParameter(a: A): PipelineParameter }

implicit val apiTo: ToPipelineParamter[PipelinesApiInput] = new ToPipelineParamter[PipelinesApiInput] { def toGooglePipelineParameter(a: PipelinesApiInput) = ...
//repeat for PipelinesApiFileInput, PipelinesApiLiteralInput, etc.
```

(Acknowledging this is basically pre-existing)